doc/build-freebsd.md

```diff
@@ -4,35 +4,6 @@
 
 This guide describes how to build bitcoind, command-line utilities, and GUI on FreeBSD.
 
-## Dependencies
-
-The following dependencies are **required**:
-
- Library                                                               | Purpose    | Description
- ----------------------------------------------------------------------|------------|----------------------
- [autoconf](https://svnweb.freebsd.org/ports/head/devel/autoconf/)     | Build      | Automatically configure software source code
- [automake](https://svnweb.freebsd.org/ports/head/devel/automake/)     | Build      | Generate makefile (requires autoconf)
- [libtool](https://svnweb.freebsd.org/ports/head/devel/libtool/)       | Build      | Shared library support
- [pkgconf](https://svnweb.freebsd.org/ports/head/devel/pkgconf/)       | Build      | Configure compiler and linker flags
- [git](https://svnweb.freebsd.org/ports/head/devel/git/)               | Clone      | Version control system
- [gmake](https://svnweb.freebsd.org/ports/head/devel/gmake/)           | Compile    | Generate executables
- [boost-libs](https://svnweb.freebsd.org/ports/head/devel/boost-libs/) | Utility    | Library for threading, data structures, etc
- [libevent](https://svnweb.freebsd.org/ports/head/devel/libevent/)     | Networking | OS independent asynchronous networking
-
-
-The following dependencies are **optional**:
-
-  Library                                                                    | Purpose          | Description
-  ---------------------------------------------------------------------------|------------------|----------------------
-  [db5](https://svnweb.freebsd.org/ports/head/databases/db5/)                | Berkeley DB      | Wallet storage (only needed when wallet enabled)
-  [qt5](https://svnweb.freebsd.org/ports/head/devel/qt5/)                    | GUI              | GUI toolkit (only needed when GUI enabled)
-  [libqrencode](https://svnweb.freebsd.org/ports/head/graphics/libqrencode/) | QR codes in GUI  | Generating QR codes (only needed when GUI enabled)
-  [libzmq4](https://svnweb.freebsd.org/ports/head/net/libzmq4/)              | ZMQ notification | Allows generating ZMQ notifications (requires ZMQ version >= 4.0.0)
```

Comment:
I think it is useful to know the name of the optional dependency. Maybe add a new line on how to install all the optional dependencies?

```
pkg install libzmq4 ...

Reply:
I've added a line for libzmq4, all of the others are already mentioned.